RELEASE/scripts/autoscend.ash

```diff
@@ -2300,57 +2300,75 @@ boolean questOverride()
 
 boolean L13_towerNSNagamar()
 {
+	// the first if check will skip getting a wand if autoscend configuration says we don't want one AND you are not on step12 of the quest
+	// if you are on step12 it will override the configuration and proceed to get a wand anyways
+	// quest step12 means you fought the sorceress and lost due to not having a wand.
+	// autoscend only reaches step12 of the quest if autoscend was incapable of acquiring a wand before the sorceress
+	// it then has to fallback to bear verb orgy, which itself cannot be done until step12
+	
 	if (!get_property("auto_wandOfNagamar").to_boolean() && internalQuestStatus("questL13Final") != 12)
 	{
 		return false;
 	}
-
+	
 	if(item_amount($item[Wand of Nagamar]) > 0)
 	{
 		set_property("auto_wandOfNagamar", false);
 		return false;
 	}
-	else if (internalQuestStatus("questL13Final") == 12)
-	{
-		return autoAdv($location[The VERY Unquiet Garves]);
-	}
-	else if(pulls_remaining() >= 2)
+	
+	if(in_koe() && item_amount($item[Wand of Nagamar]) >= 30)
```

Comment:
I think you meant to use `$item[rare Meat Isotope]` here.

Reply:
I did, thank you. fixed on commit 0b0478e